android-core/src/main/java/com/mparticle/internal/MPUtility.java

```diff
@@ -70,6 +70,7 @@ public class MPUtility {
     private static String sOpenUDID;
     private static final char[] HEX_CHARS = "0123456789abcdef".toCharArray();
     private static final String TAG = MPUtility.class.toString();
+    private static AdIdInfo infoId = null;
```

Comment:
```suggestion
    private static AdIdInfo adInfoId = null;
```
A nitpick, but maybe this should be named `adInfoId` for clarity as `infoId` is fairly generic and this utility class does various things.